decoder/candidates_test.go

```diff
@@ -1013,7 +1015,7 @@ func TestDecoder_CandidatesAtPos_multipleTypes(t *testing.T) {
 	expectedCandidates := lang.CompleteCandidates([]lang.Candidate{
 		{
 			Label:  "for_each",
-			Detail: "Optional, set of dynamic or map of dynamic",
+			Detail: "Optional, set of any single type or map of any single type",
```

Comment:
Just improving the experience here as this is presumably more correct and easier to understand for the end-user.